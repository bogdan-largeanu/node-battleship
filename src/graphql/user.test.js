describe('GraphQL Type: User', () => {
    it(`fetch user's only scalar values`, async () => {
        const { data } = await query(
            {
                query: `
                {
                    user(id: 1) {
                        id
                        email
                    }
                }`
            }
        );

        expect(data).toMatchSnapshot();
    });

    it(`fetch user's friend's scalar values`, async () => {
        const { data } = await query(
            {
                query: `
                {
                    user(id: 1) {
                        friends {
                            id
                            email
                        }
                    }
                }`
            }
        );

        expect(data).toMatchSnapshot();
    });
});
