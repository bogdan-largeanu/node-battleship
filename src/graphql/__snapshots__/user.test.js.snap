// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GraphQL Type: User fetch user's friend's scalar values 1`] = `
Object {
  "user": Object {
    "friends": Array [],
  },
}
`;

exports[`GraphQL Type: User fetch user's only scalar values 1`] = `
Object {
  "user": Object {
    "email": "user@example.com",
    "id": "1",
  },
}
`;
